src/tower.ts

```diff
@@ -17,7 +17,7 @@ export class PisaTower {
     }
 
     configs: {
-        [type: number]: IChannelConfig<EthereumAppointment, Inspector<EthereumAppointment>>;
+        [type: string]: IChannelConfig<EthereumAppointment, Inspector<EthereumAppointment>>;
```

Comment:
The type was incorrectly declared as `number` rather than the `string`.

Reply:
Good catch!